models/article.model.js

```diff
@@ -65,3 +65,28 @@ exports.selectComments = (id) => {
       return rows;
     });
 };
+
+exports.selectPatchVotesArticle = (id, patch) => {
+  if (!patch || !patch.inc_votes) {
+    return Promise.reject({ status: 400, msg: "Invalid Request" });
+  }
+  return db
+    .query("SELECT * FROM articles WHERE article_id = $1;", [id])
```

Comment:
Is it necessary to check the id exists with a separate query?

Reply:
the main reason for that query is to grab votes from articles and then later to add to that new vote. and on top of that I could check if id exist, so don't understand what is wrong with this approach. I need to grab votes from articles somehow.

Reply:
ok. changed so there is only one db query. thank you for feedback.